tests/libres_tests/res/job_queue/test_job_queue.py

```diff
@@ -198,16 +200,37 @@ def callback(arg):
             for job in job_queue.job_list:
                 job.wait_for()
 
+    async def test_retry_on_closed_connection(self):
```

Comment:
The test isn't completed, it was mostly just a draft in the beginning. I hope it will be more obvious once it's finished.

Nice idea, I'll try it out!

Reply:
Ah, I was a bit too eager! Sorry :)